### ISSUE TITLE
A11y_Visual Studio Code Services_Home_Header_Non Text Contrast: Color contrast ratio between the keyboard focus and the background for the items which are present in the auto suggestion is 1.224:1 which is less than 3:1.

### ISSUE THREAD

**Issue by Samhitha66** [microsoft/vscode-docs#6197]
### GitHubTags

#WCAG1.4.11; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #Chrome; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_Apr2023; #Non Text Contrast; #Regressed:04-13-23;

### Environment Details

Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Chrome Version 111.0.5563.147 (Official Build) (64-bit)

### Repro Steps

1. Hit the application URL: https://code.visualstudio.com/
2. Tab till "Search" field and type anything. (Auto suggestion will appear)
3. Tab till auto suggestion.
4. Open AI4D. 
5. Tab till Color contrast analyzer. (Select color 1 and color 2 as background and focus)
6. Verify whether the color contrast ratio between the keyboard focus and the background for the items which are present in the auto suggestion is greater than or equal to 3:1. or not.

### Actual Result
Color contrast ratio between the keyboard focus and the background for the items which are present in the auto suggestion is 1.224:1 which is less than 3:1.

### Expected Result
Color contrast ratio between the keyboard focus and the background for the items which are present in the auto suggestion should be greater than or equal to 3:1.

### User Impact
Low vision users will not be able to identify where exactly the keyboard focus is present in auto suggestion.

### Attachment
![Bug_Home_Header_Non Text Contrast](https://user-images.githubusercontent.com/99243079/229737115-1acb3598-5a15-469a-8565-dc56870b70fc.png)

**Comment by rzhao271** [comment]
Hi @Samhitha66, I don't think we control the behaviour of that popup; the popup is specific to Google Chrome. Are you able to reproduce the issue using Microsoft Edge or Safari?

**Comment by Samhitha66** [comment]
@rzhao271 
In microsoft edge they are provided the border for the focused element.
Please refer the below attachment
![image](https://user-images.githubusercontent.com/99243079/231674331-648b8377-4d6f-46ce-863d-8bcb8aa972d6.png)

**Comment by rzhao271** [comment]
Closing as a wontfix because:
1. The issue occurs on Chrome only.
2. We don't control the suggestions UX rendered by Chrome, including the colours used for it.

**Comment by Samhitha66** [comment]
@rzhao271 
**Please do not close this bug.**
As per the above comment your team don't control the suggestions UX rendered by Chrome.
Is this external to any other team. If yes please let us know. So that we will route this bug to respective team.
Thanks!!
#Regressed:04-21-23;

**Comment by rzhao271** [comment]
Thanks. I have filed the upstream Chromium issue at https://bugs.chromium.org/p/chromium/issues/detail?id=1435556.

**Comment by Samhitha66** [comment]
@rzhao271 
As per the above comment its external to chrome.
External tracking ID: https://bugs.chromium.org/p/chromium/issues/detail?id=1435556
We will take action on this bug once the external bug is fixed and closed.
#Regressed:04-24-23; #External; #External:Chrome; #ThirdParty; #eDAD:3P:1435556;

**Comment by rzhao271** [comment]
Hi @Samhitha66, I'm wondering whether this bug can be downgraded to a Sev3?
It is currently due next month for our team, but the issue is external to the site itself, and has to do with the browser the end user is using. Google Chrome also has their own release cadence, and so it could take weeks to months for the fix to be released to the stable version of the browser.

**Comment by Samhitha66** [comment]
@rzhao271 
We can't able to downgrade the severity as the ratio is too less than 3:1. 
Please provide us the ETA when they are going to fix. So that we can tag the bug accordingly.
Thanks!!
#Regressed:05-10-23;

**Comment by rzhao271** [comment]
@Samhitha66 upstream has marked the bug as a wontfix (https://bugs.chromium.org/p/chromium/issues/detail?id=1435556#c9). We're unable to proceed with this issue. Please suggest next steps.

**Comment by rzhao271** [comment]
Closing as a wontfix due to upstream marking the issue as a wontfix and due to lack of a reply.

**Comment by Samhitha66** [comment]
@rzhao271 
Sorry for the late reply.
Is this behavior due to HTML component?
Please reopen this issue by providing your input.
Thanks!!

**Comment by rzhao271** [comment]
@Samhitha66 we have an SLA on this issue because you labelled it as a sev 2 and did not downgrade it. Reopening the issue could cause us to exceed a deadline outside of our control.

The behaviour you're seeing is a browser issue. It does not happen with Microsoft Edge or Safari. It does not have to do with how we implemented the website.

**Comment by Samhitha66** [comment]
As per discussion with SME , we are not supposed to check contrast ratio for auto suggestions.
Hence closing this bug.
Thanks!!
#Regressed:05-26-23;
#Closed;
